Probe for pulseaudio first otherwise we might end up talking to
pulseaudio via the alsa wrapper which doesn't work well.
Index: openal-soft-1.9.563/Alc/ALc.c
===================================================================
--- openal-soft-1.9.563.orig/Alc/ALc.c
+++ openal-soft-1.9.563/Alc/ALc.c
@@ -48,6 +48,9 @@ typedef struct BackendInfo {
     BackendFuncs Funcs;
 } BackendInfo;
 static BackendInfo BackendList[] = {
+#ifdef HAVE_PULSEAUDIO
+    { "pulse", alc_pulse_init, alc_pulse_deinit, alc_pulse_probe, EmptyFuncs },
+#endif
 #ifdef HAVE_ALSA
     { "alsa", alc_alsa_init, alc_alsa_deinit, alc_alsa_probe, EmptyFuncs },
 #endif
@@ -66,9 +69,6 @@ static BackendInfo BackendList[] = {
 #ifdef HAVE_PORTAUDIO
     { "port", alc_pa_init, alc_pa_deinit, alc_pa_probe, EmptyFuncs },
 #endif
-#ifdef HAVE_PULSEAUDIO
-    { "pulse", alc_pulse_init, alc_pulse_deinit, alc_pulse_probe, EmptyFuncs },
-#endif
 
     { "wave", alc_wave_init, alc_wave_deinit, alc_wave_probe, EmptyFuncs },
 
